package.json

```diff
@@ -37,9 +39,18 @@
     "json": "^9.0.4",
     "json-loader": "0.5.4",
     "localforage": "1.5.0",
+    "semantic-release": "^6.3.2",
     "tap": "8.0.1",
     "text-encoding": "0.6.4",
     "travis-after-all": "^1.4.4",
     "webpack": "2.2.1"
+  },
+  "config": {
+    "commitizen": {
+      "path": "./node_modules/cz-conventional-changelog"
+    }
+  },
+  "release": {
+    "branch": "develop"
```

Comment:
At some point in the future we may want to change this to "master" (or remove it, which is equivalent) but for now I think continuing to publish from "develop" makes sense.